packaging/docker/run.sh

```diff
@@ -13,6 +13,23 @@ if [ ! "${DO_NOT_TRACK:-0}" -eq 0 ] || [ -n "$DO_NOT_TRACK" ]; then
   touch /etc/netdata/.opt-out-from-anonymous-statistics
 fi
 
+
+BALENA_PGID=$(ls -nd /var/run/balena.sock | awk '{print $4}')
+DOCKER_PGID=$(ls -nd /var/run/docker.sock | awk '{print $4}')
+
+re='^[0-9]+$'
+if [[ $BALENA_PGID =~ $re ]]; then
+  echo "Netdata detected balena-engine.sock"
+  DOCKER_HOST='/var/run/balena-engine.sock'
+  PGID="$BALENA_PGID"
+else if [[ $DOCKER_PGID =~ $re ]]; then
```

Comment:
This needs to be `elif` not `else if`. Fixing that should resolve most of the shellcheck errors.